runtime/mandala/src/lib.rs

```diff
@@ -1162,7 +1163,12 @@ impl module_transaction_pause::Config for Runtime {
 
 parameter_types! {
 	// Sort by fee charge order
-	pub DefaultFeeSwapPathList: Vec<Vec<CurrencyId>> = vec![vec![AUSD, ACA], vec![AUSD, LDOT, ACA], vec![AUSD, DOT, ACA], vec![AUSD, RENBTC, ACA]];
+	pub DefaultFeeSwapPathList: Vec<Vec<CurrencyId>> = vec![
+		vec![AUSD, DOT, ACA],
+		vec![DOT, ACA],
+		vec![LDOT, DOT, ACA],
+		vec![RENBTC, AUSD, ACA]
```

Comment:
should this be `[RENBTC, AUSD, DOT, ACA]`?

Reply:
https://github.com/AcalaNetwork/Acala/blob/c34d941c09c9a3d646f013bdf0d8087dd8932ae4/runtime/mandala/src/lib.rs#L1109-L1115

Here have AUSD/ACA trading pairs

Reply:
if we already have AUSD/ACA, I think the first element be `[AUSD, ACA]` would better, isn't it? but of course `[AUSD, DOT, ACA]` works too.

Reply:
Yeah. This is adjusting for the integration test.

https://github.com/AcalaNetwork/Acala/blob/df1daf6e747a157d8022eab358468a348b3240c3/runtime/integration-tests/src/payment.rs#L124-L141